pkg/apis/submariner.io/v1/endpoint.go

```diff
@@ -0,0 +1,45 @@
+package v1
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+)
+
+func (ep *EndpointSpec) GetBackendPortStr(configName, defaultValue string) (string, error) {
+	if portStr := ep.BackendConfig[configName]; portStr != "" {
+		port, err := parsePort(portStr)
+		if err != nil {
+			return defaultValue, errors.Wrapf(err, "error parsing backend config %s", configName)
+		}
+
+		return strconv.Itoa(int(port)), nil
+	}
+
```

Comment:
This could call `GetBackendPort` but I would suggest removing this function. libreswan can simply call `GetBackendPort` and convert to string.

Reply:
Done